.github/workflows/compute.yml

```diff
@@ -35,12 +35,11 @@ jobs:
         with:
           node-version: "20.10.0"
 
-      - name: install dependencies
-        run: yarn
-
       - name: execute directive
-        run: npx tsx ./src/main.ts
-        id: plugin-name
+        run: node ./dist/index.js
```

Comment:
This doesn't look right. Run ts directly

Reply:
this should summon the `action.yml` file located at the root of the repo.

Reply:
What do you mean summon? There isn't an action.yml file at the base, I can create one but I don't get what you meant by summon

Reply:
An `action.yml` similar of https://github.com/ubiquity-os-marketplace/text-conversation-rewards/blob/development/action.yml should be at the root of the repo, and triggered by this action file.

Reply:
Actually I was always wondering what is the benefit of `action.yml` instead of just calling `node ./dist/index.js` directly. It seems unnecessary and you have to repeat the inputs

Reply:
I'll await further inputs before doing anything because I knew there might be some conflicts

Reply:
> Actually I was always wondering what is the benefit of `action.yml` instead of just calling `node ./dist/index.js` directly. It seems unnecessary and you have to repeat the inputs

With action.yml we can call it directly from GitHub actions without the kernel as far as I'm aware. It seems like a small step for extended compatibility.